test/Microsoft.TemplateEngine.Core.UnitTests/ConditionalTests.cs

```diff
@@ -57,16 +57,25 @@ private IOperationProvider[] XmlStyleCommentConditionalsOperations
         {
             get
             {
+                // This is the operationId (flag) for the balanced nesting
+                string commentFixingOperationId = "Fix pseudo comments";
+
+                // This is not an operationId (flag), it does not toggle the operation.
+                // But conditional doesn't care, it takes the flags its given and sets them as appropriate.
+                // It lets BalanceNesting know it's been reset
+                string commentFixingResetId = "Reset pseudo comment fixer";
```

Comment:
We should reuse (copy) these helpers from the tests in the config reader for making the default XML/HTML/Razor style directives